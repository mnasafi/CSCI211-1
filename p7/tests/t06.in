echo t06: Insert days of week into tree, then call find on strings that should be in the tree.
insert Saturday
insert Friday
insert Tuesday
insert Monday
insert Thursday
insert Wednesday
insert Sunday
echo Calling find on <Monday> (should be in tree):
find Monday
echo Calling find on <Tuesday> (should be in tree):
find Tuesday
echo Calling find on <Wednesday> (should be in tree):
find Wednesday
echo Calling find on <Thursday> (should be in tree):
find Thursday
echo Calling find on <Friday> (should be in tree):
find Friday
echo Calling find on <Saturday> (should be in tree):
find Saturday
echo Calling find on <Sunday> (should be in tree):
find Sunday
