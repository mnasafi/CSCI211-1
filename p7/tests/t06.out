t06: Insert days of week into tree, then call find on strings that should be in the tree.
Calling find on <Monday> (should be in tree):
<Monday> is in tree.
Calling find on <Tuesday> (should be in tree):
<Tuesday> is in tree.
Calling find on <Wednesday> (should be in tree):
<Wednesday> is in tree.
Calling find on <Thursday> (should be in tree):
<Thursday> is in tree.
Calling find on <Friday> (should be in tree):
<Friday> is in tree.
Calling find on <Saturday> (should be in tree):
<Saturday> is in tree.
Calling find on <Sunday> (should be in tree):
<Sunday> is in tree.
